frontend/src/components/nlp/NlpImportDialog.tsx

```diff
@@ -39,8 +39,8 @@ export const NlpImportDialog: FC<NlpImportDialogProps> = ({
       mutationFn: async (attachmentId: string | null) => {
         attachmentId && (await apiClient.importNlpSamples(attachmentId));
       },
-      onSuccess: () => {
-        queryClient.removeQueries({
+      onSuccess: async () => {
+        await queryClient.invalidateQueries({
```

Comment:
I think we should remove the queries, cause it will still fetch invalidated queries. Or maybe I'm missing something.

Reply:
When i test it with removeQueries the selected queries are removed without triggering a refreshing to update the counts counters but if we invalidate the queries selected will be forced to re-fetch and this will update the UI count numbers